src/careamics/config/architectures/lvae_model.py

```diff
@@ -160,7 +251,10 @@ def set_3D(self, is_3D: bool) -> None:
         is_3D : bool
             Whether the algorithm is 3D or not.
         """
-        raise NotImplementedError("VAE is not implemented yet.")
+        if is_3D:
+            self.conv_dims = 3
+        else:
+            self.conv_dims = 2
 
```

Comment:
We have conv strides because we can also have 2.5D which is 3D encoder 2D decoder. And we'd have to define strides anyway, otherwise we'd have to hardcode them which is not good

Reply:
Ye, I know. But I would anyway create also a `conv_dims` attributes within the model class derived from the strides